src/test/java/io/vertx/circuitbreaker/impl/CircuitBreakerMetricsTest.java

```diff
@@ -215,7 +215,7 @@ public void testLatencyComputation(TestContext tc) {
   }
 
   @Test
-  @Repeat(100)
+  @Repeat(50)
```

Comment:
Any reason to reduce the number of iteration?

Reply:
Yes, I noticed that after having added my test unit and run the test with `mvn test`, it failed with a StackOverflowException with a stack almost entirely made out of composite Futures. I took a hunch and guessed that it was caused by the high number of iterations. I re-ran the tests now and terminated successfully. I'm going to revert this commit!

Reply:
edit: re-ran the tests **with the original number of iterations**